ember-file-upload/src/index.ts

```diff
@@ -2,7 +2,7 @@ import { Queue } from './queue';
 import { UploadFile } from './upload-file';
 import FileQueueService from './services/file-queue';
 import { uploadHandler } from './mirage/upload-handler';
-import { FileSource, FileState } from './interfaces';
+import { FileSource, FileState, QueueName } from './interfaces';
```

Comment:
I think this should work.

Tested locally by linking a local build of `ember-file-upload` with this change into an test ember app with typescript.

```suggestion
import { FileSource, FileState, type QueueName } from './interfaces';
```

Reply:
@BoussonKarel Would you like to try this or shall I open a new PR?